src/models/network.js

```diff
@@ -269,6 +272,10 @@ Network.prototype.setNick = function(nick) {
 		// Case insensitive search
 		"i"
 	);
+
+	if (this.keepNick === nick) {
+		this.keepNick = null;
```

Comment:
I've not looked, but I'm presuming this is messing up your build. Yay prettier.

Reply:
Oh, hmmm. Your build is passing. I wonder why this is messed up then, I wonder if it's just github UI getting confused with something.

Reply:
Oh. It's refined github messing it up. Ignore me.